src/uzal_cost.jl

```diff
@@ -115,6 +119,112 @@ function uzal_cost(Y::Dataset ; Tw::Int = 40, K::Int = 3, w::Int = 1, samplesize
     return L
 end
 
+"""
+    uzal_cost_local(Y::Dataset; kwargs...) → L_local
```

Comment:
can't we just point to the documentation of `uzal_cost`, and only say here what is different with respect to `uzal_cost`, instead of repeating the whole docstring?

Reply:
Of course

Reply:
Is there a proper way of "pointing"?

Reply:
Yes! You can use this syntax: 
```
[`uzal_cost`](@ref)
```
which will create a corssreferenced link automatically to the existing docstrign.

Reply:
Will do!